lib/digitalocean/charts/q-ingress-tls/templates/ingress-customdns.j2.yaml

```diff
@@ -0,0 +1,43 @@
+{%- if routes|length >= 1  %}
+{%- for custom_domain in custom_domains %}
+{%- if custom_domain.target_domain %}
+---
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: {{ name }}-{{ custom_domain.domain_hash }}
+  namespace: {{ namespace }}
+  labels:
+    ownerId: {{ owner_id }}
+    routerName: {{ name }}
+    routerId: {{ id }}
+    envId: {{ environment_id }}
+    fqdn: "{{ custom_domain.domain }}"
+    fqdnId: {{ custom_domain.domain_hash }}
+  annotations:
+    ingress.kubernetes.io/ssl-redirect: "true"
+    kubernetes.io/tls-acme: "true"
+    cert-manager.io/issuer: "{{ custom_domain.domain_hash }}"
+    kubernetes.io/ingress.class: "{{ id }}"
+    # this domain should be CNAME-ed by the client. ex: app.foo.bar -> app.qovery.io
+    external-dns.alpha.kubernetes.io/hostname: {{ custom_domain.target_domain }}
+    external-dns.alpha.kubernetes.io/target: {{ nlb_ingress_hostname }}
```

Comment:
will this work, with "nlb"?

Reply:
it's a context variable, in fact it's a ingress_ip but yes, i remove it